structure_tests/structure_test_v1.go

```diff
@@ -107,3 +120,45 @@ func (st StructureTestv1) RunFileContentTests(t *testing.T) {
 		}
 	}
 }
+
+// Given a list of environment variables and a list of lists of strings,
+// retrieve each environment variable's value and replace all occurrences
+// of it in each list of strings provided.
+func SubstituteEnvVars(t *testing.T, vars []string, lists ...*[]string) {
```

Comment:
I don't quite understand how this is supposed to be used. Could you add some docs with examples?

At first I thought we should just set the env var before running a command, but it looks like we're replacing it in the stdout/stderr?